agent/engine/execcmd/manager_unsupported.go

```diff
@@ -22,6 +22,11 @@ import (
 	"github.com/aws/amazon-ecs-agent/agent/dockerclient/dockerapi"
 )
 
+const (
+	// HostLogDir in unsupported platforms for exec is empty
```

Comment:
nit: this is a bit self-explanatory, maybe this comment is not needed, or use this comment to explain what would happen later if we use empty string here

Reply:
added more explanation.